Makefile

```diff
@@ -3,21 +3,41 @@ GOPATH ?= `go env GOPATH`
 GOBIN ?= $(GOPATH)/bin
 GO111MODULE = auto
 
-CONTAINER_CMD ?= $(shell command -v podman)
+
+CONTAINER_RUNTIME ?=
+
+ifeq ($(CONTAINER_RUNTIME),)
+CONTAINER_CMD ?= $(shell type -P podman)
 ifeq ($(CONTAINER_CMD),)
-CONTAINER_CMD := $(shell command -v docker)
+CONTAINER_CMD := $(shell type -P docker)
+endif
+CONTAINER_RUNTIME=$(shell basename $(CONTAINER_CMD))
+else
+CONTAINER_CMD := $(shell type -P $(CONTAINER_RUNTIME))
 endif
 
 REGISTRY ?= quay.io
-REGISTRY_ACCOUNT ?= kubev2v
+# TODO remove REGISTRY_ORG check once the changes are merged in forkliftci
```

Comment:
were the changes posted or merged to forkliftci?

Reply:
Posted them today: kubev2v/forkliftci#106